jukebox/src/lib.rs

```diff
@@ -28,11 +25,13 @@ pub struct Jukebox {
     iso_path: String,
     _output_stream: OutputStream,
     sink: Sink,
-    get_dolphin_volume_fn: ForeignGetVolumeFn,
+    dolphin_system_volume: f32,
+    dolphin_music_volume: f32,
+    melee_volume: f32,
```

Comment:
It might make sense to rename a few symbols for consistency:
- `Jukebox::melee_volume` -> `Jukebox::melee_music_volume`
- `slprs_exi_device_jukebox_set_music_volume` -> `slprs_exi_device_jukebox_set_melee_music_volume`
- `SlippiEXIDevice::jukebox_set_music_volume` -> `SlippiEXIDevice::jukebox_set_melee_music_volume`